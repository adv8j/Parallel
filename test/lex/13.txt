@Task
{
    .ct <   -
    5;
    .wt{someIdentifier, 56};
}
