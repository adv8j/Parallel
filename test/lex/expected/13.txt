TASK
LBRACE
TASK_CHANNEL
LT
MINUS
INT_LITERAL
SEMICOLON
CHANNEL_WAIT
LBRACE
IDENTIFIER
COMMA
INT_LITERAL
RBRACE
SEMICOLON
RBRACE
